awx/main/expect/isolated_manager.py

```diff
@@ -407,46 +407,50 @@ def health_check(cls, instance_qs, awx_application_version):
         args = cls._build_args('heartbeat_isolated.yml', hostname_string)
         args.extend(['--forks', str(len(instance_qs))])
         env = cls._base_management_env()
-        env['ANSIBLE_STDOUT_CALLBACK'] = 'json'
```

Comment:
Embedding this data in stdout and relying on stdout to _always_ be parseable JSON isn't robust enough; it's pretty easy to break this w/ a number of `ansible.cfg` settings that affect output/display.